lib/ansible/plugins/callback/default.py

```diff
@@ -75,9 +75,16 @@ def set_options(self, task_keys=None, var_options=None, direct=None):
                 value = constant
             setattr(self, option, value)
 
+    def _host_spec(self, result):
+        hostname = result._host.get_name()
+        delegated_vars = result._result.get('_ansible_delegated_vars', None)
+        if delegated_vars:
+            return "%s -> %s" % (hostname, delegated_vars['ansible_host'])
+        return "%s" % (hostname,)
+
```

Comment:
tempted to move to base class and make static

Reply:
Sure, I was already leaning to `@staticmethod`. Do you have a preference for `_host_spec()`, vs `host_spec()`, vs e.g. `host_label()`. I'll can also add a test, particularly if it's to become a supported API.

Reply:
I like host_label()